src/sections/Projects/Project-grid/projectGrid.style.js

```diff
@@ -1,478 +1,486 @@
 import styled from "styled-components";
+//These are the diferent breakpoints used in this file for different screen sizes.
+import { breakpoints } from "../../../theme/modern/themeStyles";
 export const ProjectWrapper = styled.div`
-    padding-bottom: 5rem;
-    h3{
-        color: ${props => props.theme.whiteSixToGreen3C494F};  
-        margin: 0;
-        padding: 0;
-        font-weight: 500;
-        text-align:center;
-        transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
+  padding: 0 2rem;
+  padding-bottom: 5rem;
+  font-size: 1rem;
```

Comment:
Is the specification of font-size here necessary?

Reply:
Yes I think so because I changed all the px to rem considering the font size as base, but I think nothing will change if we remove it

Reply:
Please remove, if not necessary.

Reply:
Done.